crates/test/cometbft-launcher/src/lib.rs

```diff
@@ -0,0 +1,336 @@
+//! Launch cometbft as a subprocess for running tests.
+//!
+//! # Examples
+//! ```no_run
+//! # tokio_test::block_on(async {
+//! use cometbft_launcher::CometBft;
+//! let cometbft = CometBft::builder()
+//!     .proxy_app("tcp://127.0.0.1:26657")
```

Comment:
```suggestion
//!     .proxy_app("tcp://127.0.0.1:26658")
```

the app default port is at 26658